doc/release-notes-25527.md

```diff
@@ -0,0 +1,8 @@
+Updated settings
+----------------
+
+- Specifying a `-maxsigcachesize` MiB value which is too large will now result
```

Comment:
`-maxsigcachesize` is DEBUG_ONLY, should we cover these in release notes too?

Reply:
Very good point! Fixed in f66cd5b3aa